php-classes/Slate/CBL/Calculators/Growth/PerformanceLevelMinusBaseline.php

```diff
@@ -8,7 +8,13 @@ class PerformanceLevelMinusBaseline implements IGrowthCalculator
 {
     public static function calculateGrowth(StudentCompetency $StudentCompetency)
     {
-        $baseline = $StudentCompetency->BaselineRating;
+        $baseline = round($StudentCompetency->BaselineRating, 1);
```

Comment:
@nbey let's create a `getBaselineRating()` getter method and have it apply rounding according to a public static precision config on the `StudentCompetency` class

Reply:
should we look to replace instances of `StudentComptency->BaselineRating` -> `StudentCompetency->getBaselineRating()` in other places as well?